packages/workspace/src/browser/workspace-uri-contribution.ts

```diff
@@ -86,13 +106,11 @@ export class WorkspaceUriLabelProviderContribution extends DefaultUriLabelProvid
      */
     getLongName(element: URI | FileStat): string {
         const uri = this.getUri(element);
```

Comment:
it should be:
```ts
const uri = this.getUri(element);
return uri ? this.workspaceService.getWorkspaceRootUri(uri) : super.getLongName(uri)
```

After https://github.com/theia-ide/theia/pull/3613/files#diff-70a982fe47d80b1a0ac76c9c84fd5a4b gets merged.